RNTester/js/components/RNTesterExampleList.js

```diff
@@ -73,8 +73,8 @@ class RowComponent extends React.PureComponent<{
               onPress={this._onPress}>
               <View
                 style={[
-                  rowStyle, // TODO(macOS ISS#2323203)
                   {backgroundColor: theme.SystemBackgroundColor},
```

Comment:
So are the right colours brought in by `theme.SystemBackgroundColor`?

Reply:
The list has properly being changed when switching dark mode on/off so I guess so? Because it worked as expected to me, I didn't really investigated what's under `theme.SystemBackgroundColor` really

Reply:
Fair enough 👍